dataland-frontend/tests/component/router/Router.cy.ts

```diff
@@ -2,13 +2,11 @@ import { routes } from "@/router";
 import { shallowMount, mount } from "@vue/test-utils";
 import App from "@/App.vue";
 import { createRouter, createWebHistory, Router } from "vue-router";
-import { expect } from "@jest/globals";
-import { getRequiredPlugins } from "../TestUtils";
 
 function mountAppWithRouter(routerToBeUsedWithMount: Router) {
   return mount(App, {
```

Comment:
Darüber hinaus kann ich in der IDE keine Component Tests mehr einzeln über das play-Symbol links triggern.